code/data_structures/avl_tree/avl_tree.java

```diff
@@ -234,10 +254,26 @@ public static void main(String[] args)
         System.out.println("Preorder traversal of "+
                             "constructed tree is : ");
         tree.preOrder(tree.root);
+		System.out.println("");
```

Comment:
Do not indent unless necessary. For separation, put a newline in between each section.